tests/test_wien2kparser.py

```diff
@@ -121,3 +124,16 @@ def test_dos(parser, caplog):
     assert sec_dos_up.species_projected[1].value[926].magnitude == approx(
         2.7395685667470246e17
     )
+
+def test_core_hole(parser, caplog):
+    archive = EntryArchive()
+    parser.parse('tests/data/wien2k/TiN-corehole/TiN-corehole.scf', archive, None)
+
+    sec_method = archive.run[0].method[0]
+    atom_par = sec_method.atom_parameters[0]
+    assert atom_par.atom_index == 2
+    assert atom_par.core_hole.n_electrons_excited == 1
```

Comment:
This one is actually a float (although this is kinda not really physically intuitive to speak about half-electron :smile: )

Reply:
I agree with you but this is actually in definition, too. 

https://nomad-lab.eu/prod/v1/gui/analyze/metainfo/nomad/section_definitions@nomad.datamodel.metainfo.simulation.method.AtomParameters/core_hole/n_electrons_excited

Should I change the definition of n_electrons_excited?

Reply:
This is what I get when I follow your link... 
![image](https://github.com/user-attachments/assets/25cd9898-eb18-4e83-a340-bcfaa63c392f)

`type: float64 (numpy)` but maybe you are refering to something else?

Reply:
> This one is actually a float (although this is kinda not really physically intuitive to speak about half-electron 😄 )

I am not sure what you mean. Would you like to change the assert like this: 
`assert atom_par.core_hole.n_electrons_excited == 1.0`?

Reply:
Yeah, that would be way better, but I guess this is probably safe in python as well :shrug:

Reply:
Anyway this is nitpicking at this point...

Reply:
No problem. Done.